src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -907,98 +903,26 @@ void IrcMessageHandler::handleWhisperMessage(Communi::IrcMessage *message)
 
-    if (_message->flags.has(MessageFlag::ShowInMentions))
-    {
-        getApp()->twitch->mentionsChannel->addMessage(_message);
-    }
-
-    c->addMessage(_message);
-
-    auto overrideFlags = std::optional<MessageFlags>(_message->flags);
-    overrideFlags->set(MessageFlag::DoNotTriggerNotification);
-    overrideFlags->set(MessageFlag::DoNotLog);
-
-    if (getSettings()->inlineWhispers &&
-        !(getSettings()->streamerModeSuppressInlineWhispers &&
-          isInStreamerMode()))
-    {
-        getApp()->twitch->forEachChannel(
-            [&_message, overrideFlags](ChannelPtr channel) {
-                channel->addMessage(_message, overrideFlags);
-            });
-    }
-}
-
-std::vector<MessagePtr> IrcMessageHandler::parseUserNoticeMessage(
-    Channel *channel, Communi::IrcMessage *message)
-{
-    std::vector<MessagePtr> builtMessages;
-
-    auto tags = message->tags();
-    auto parameters = message->parameters();
-
-    QString msgType = tags.value("msg-id").toString();
-    QString content;
-    if (parameters.size() >= 2)
-    {
-        content = parameters[1];
-    }
-
-    if (isIgnoredMessage({
-            .message = content,
-            .twitchUserID = tags.value("user-id").toString(),
-            .isMod = channel->isMod(),
-            .isBroadcaster = channel->isBroadcaster(),
-        }))
-    {
-        return {};
-    }
-
-    if (specialMessageTypes.contains(msgType))
-    {
-        // Messages are not required, so they might be empty
-        if (!content.isEmpty())
-        {
-            MessageParseArgs args;
-            args.trimSubscriberUsername = true;
-
-            TwitchMessageBuilder builder(channel, message, args, content,
-                                         false);
-            builder->flags.set(MessageFlag::Subscription);
-            builder->flags.unset(MessageFlag::Highlighted);
-            builtMessages.emplace_back(builder.build());
-        }
-    }
-
-    auto it = tags.find("system-msg");
-
-    if (it != tags.end())
-    {
-        // By default, we return value of system-msg tag
-        QString messageText = it.value().toString();
-
-        if (msgType == "bitsbadgetier")
-        {
-            messageText =
-                QString("%1 just earned a new %2 Bits badge!")
-                    .arg(tags.value("display-name").toString(),
-                         kFormatNumbers(
-                             tags.value("msg-param-threshold").toInt()));
-        }
-        else if (msgType == "announcement")
-        {
-            messageText = "Announcement";
-        }
-
-        auto b = MessageBuilder(systemMessage, parseTagString(messageText),
-                                calculateMessageTime(message).time());
-
-        b->flags.set(MessageFlag::Subscription);
-        auto newMessage = b.release();
-        builtMessages.emplace_back(newMessage);
+    if (_message->flags.has(MessageFlag::ShowInMentions))
```

Comment:
`_message` seems a bit weird. Could this be renamed to something like `msg`, `built`, `builtMessage`, `whisper` or similar?

Reply:
Resolved in [`b772e3d` (#4927)](https://github.com/Chatterino/chatterino2/pull/4927/commits/b772e3dfd3ebaad21bd1111841297185d57bb479)

I opted to rename the parameter from `message` to `ircMessage`, so this function variable could be renamed to `message`